introduction.md

```diff
@@ -1,2 +1,5 @@
 HI, we are the CI team from MSTR dev ops
 My favorite code school path is Electives
+* Git
+* Javascript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?